tests/run-tests.sh

```diff
@@ -19,8 +19,9 @@ function test_component {
     case $component in
         check_impi_2021) verify_impi_2021_installation;;
         check_impi_2018) verify_impi_2018_installation;;
-        check_cuda) verify_cuda_installation;;
+        check_cuda) verify_gdrcopy_installation;verify_cuda_installation;;
```

Comment:
verify_gdrcopy_installation is called from verify_common_components. I you want it to be a switchable component, you'll need to remove it from there. Preferably it would have its own switch case here and be added to the NVIDIA configurations

Reply:
I prefer not to touch the nvidia parts as much as possible.